src/parser/Parser.ts

```diff
@@ -460,6 +463,12 @@ export class Parser {
             parentClassName,
             this.currentNamespaceName
         );
+
+        //attach annotations to statements
+        if (classAnnotations?.length) {
```

Comment:
While this technically works due to javascripts truthy/falsey, let's check for length > 0

Reply:
I got this from other code in the project.. I didn't like it; but I thought it was idiomatic

Reply:
Test should be 
```
if (classAnnotations.length > 0) {
```
and we can merge - I'll complete some other things.